account_product_fiscal_classification/tests/tests.py

```diff
@@ -138,3 +144,12 @@ def test_06_unlink_fiscal_classification(self):
             [self.classification_1_id])
         with self.assertRaises(ValidationError):
             classification.unlink()
+
+    def test_07_classification_generate_from_template(self):
```

Comment:
@danimaribeiro @legalsylvain 
In fact it seems we have some bad issue here: it seems that no test is being run at all in this module.
So look at test_01 for instance. It's suppose to call wizard method button_change_fiscal_classification:
https://github.com/akretion/account-fiscal-rule/blob/akretion-team/fiscal-classification-template/account_product_fiscal_classification/tests/tests.py#L63
but it doesn't happen!
https://coveralls.io/builds/4063409/source?filename=account_product_fiscal_classification%2Fmodels%2Fwizard_change_fiscal_classification.py#L48

Worse/ the test didn't run either on last patch by @renatonlima on Nombember 3rd:
https://coveralls.io/builds/4037079/source?filename=account_product_fiscal_classification%2Fmodel%2Fwizard_change_fiscal_classification.py#L48

I'm now wondering if we merged that module with tests simply disabled....


cc @mileo